app/http.php

```diff
@@ -156,148 +155,156 @@ function dispatch(Server $server, int $fd, int $type, $data = null): int
+                $dbForPlatform->createDocument('buckets', new Document([
+                    '$id' => ID::custom('default'),
+                    '$collection' => ID::custom('buckets'),
+                    'name' => 'Default',
+                    'maximumFileSize' => (int) System::getEnv('_APP_STORAGE_LIMIT', 0),
+                    'allowedFileExtensions' => [],
+                    'enabled' => true,
+                    'compression' => 'gzip',
+                    'encryption' => true,
+                    'antivirus' => true,
+                    'fileSecurity' => true,
+                    '$permissions' => [
+                        Permission::create(Role::any()),
+                        Permission::read(Role::any()),
+                        Permission::update(Role::any()),
+                        Permission::delete(Role::any()),
+                    ],
+                    'search' => 'buckets Default',
+                ]));
+
+                $bucket = $dbForPlatform->getDocument('buckets', 'default');
+
+                Console::info("    └── Creating files collection for default bucket...");
+                $files = $collections['buckets']['files'] ?? [];
+                if (empty($files)) {
+                    throw new Exception('Files collection is not configured.');
                 }
-                if (!$dbForProject->getCollection($key)->isEmpty()) {
-                    continue;
-                }
-
-                $attributes = \array_map(fn ($attribute) => new Document($attribute), $collection['attributes']);
-                $indexes = \array_map(fn (array $index) => new Document($index), $collection['indexes']);
-
-                Console::success('[Setup] - Creating project collection: ' . $collection['$id'] . '...');
 
-                $dbForProject->createCollection($key, $attributes, $indexes);
+                $attributes = array_map(fn ($attr) => new Document([
+                    '$id' => ID::custom($attr['$id']),
+                    'type' => $attr['type'],
+                    'size' => $attr['size'],
+                    'required' => $attr['required'],
+                    'signed' => $attr['signed'],
+                    'array' => $attr['array'],
+                    'filters' => $attr['filters'],
+                    'default' => $attr['default'] ?? null,
+                    'format' => $attr['format'] ?? ''
+                ]), $files['attributes']);
+
+                $indexes = array_map(fn ($index) => new Document([
+                    '$id' => ID::custom($index['$id']),
+                    'type' => $index['type'],
+                    'attributes' => $index['attributes'],
+                    'lengths' => $index['lengths'],
+                    'orders' => $index['orders'],
+                ]), $files['indexes']);
+
+                $dbForPlatform->createCollection('bucket_' . $bucket->getInternalId(), $attributes, $indexes);
```

Comment:
seems like a lot of unrelated changes.. can we please revert this? to avoid complexity in the review and release

Reply:
This is to simplify this and make logging easier. it's already reviewed as a part of appwrite cloud as of adding Logs DB PR and was backported here.